docs/development/writingzeppelininterpreter.md

```diff
@@ -76,20 +73,20 @@ for example,
   <value>org.apache.zeppelin.spark.SparkInterpreter,org.apache.zeppelin.spark.PySparkInterpreter,org.apache.zeppelin.spark.SparkSqlInterpreter,org.apache.zeppelin.spark.DepInterpreter,org.apache.zeppelin.markdown.Markdown,org.apache.zeppelin.shell.ShellInterpreter,org.apache.zeppelin.hive.HiveInterpreter,com.me.MyNewInterpreter</value>
 </property>
 ```
-3. start zeppelin by running ```./bin/zeppelin-deamon start```
+3. Start zeppelin by running `./bin/zeppelin-daemon.sh start`.
```

Comment:
maybe nit?: Personally I'd love to see `Zeppelin` instead of `zeppelin` since it is a brand. How about this?